phylum_project/src/lib.rs

```diff
@@ -157,7 +159,7 @@ mod tests {
         let mut config =
             ProjectConfig::new(PROJECT_ID, PROJECT_NAME.to_owned(), Some(GROUP_NAME.to_owned()));
         config.set_depfiles(vec![DepfileConfig {
-            path: PathBuf::from("Cargo.lock"),
+            path: PathBuf::from("./Cargo.lock"),
             depfile_type: "cargo".to_owned(),
         }]);
```

Comment:
Wait... Does this not work without `./` anymore? That seems like a problem...

Reply:
It works with both `Cargo.lock` and `./Cargo.lock`. It was changed in the test cases to `./Cargo.lock` to be more representative of how the config will look now (see the updated `.phylum_project` file in this PR for how `phylum init` populates it now).

Reply:
Hmm... I'm not sure when that change to `phylum init` happened.